airflow/models/xcom.py

```diff
@@ -57,14 +57,13 @@
 from airflow.utils.log.logging_mixin import LoggingMixin
 from airflow.utils.session import NEW_SESSION, provide_session
 from airflow.utils.sqlalchemy import UtcDateTime
+from airflow.utils.xcom import (
+    MAX_XCOM_SIZE,  # noqa: F401 (#30180 - leaving it for providers backward compatibility)
+    XCOM_RETURN_KEY,
+)
```

Comment:
When I tried to migrate the providers' code to `import airflow.utils.xcom`, the Airflow 2.3 compatibility test for them failed (https://github.com/apache/airflow/actions/runs/4456673101/jobs/7827374562?pr=30180) since naturally there was no corresponding package.

I'm not sure what the right way to migrate them is, would appreciate some guidance!

Reply:
Figure out a way. 

For exampele import the new constants in the old place and mark it with comments to remove it when min-airflow-version is 2.6.

The test is right (and was specifically designed to inform and prevent you from ignoribng the fact that new providers can be used with old airflow (and old providers with new airflow).

Reply:
> The test is right

Yeah, sure, I didn't have any doubts around that :)

> For exampele import the new constants in the old place and mark it with comments to remove it when min-airflow-version is 2.6.

Makes sense to me, I think this is exactly what I came up with, I'll extend the comment with the minimal airflow version according to your proposal then, thanks!